README.md

```diff
@@ -151,6 +151,8 @@ The following may be provided on a joi `.meta()` object to explicitly override d
 
 **swaggerType**: Can be used with the .any() type to add files.
 
+**schemaOverride**: A replacement Joi schema which is used to generate swagger.
```

Comment:
Could you please add a connection when this is needed, an example with usage of alternatives would be nice.